src/main/scala/tech/sourced/gemini/FeaturesHash.scala

```diff
@@ -44,10 +43,10 @@ object FeaturesHash {
     * @param docFreq
     * @return
     */
-  private def toBagOfFeatures(features: Iterable[Feature], docFreq: OrderedDocFreq): Array[Double] = {
+  def toBagOfFeatures(features: Iterable[Feature], docFreq: OrderedDocFreq): Array[Double] = {
     val OrderedDocFreq(docs, tokens, df) = docFreq
 
-    val bag = new Array[Double](tokens.size)//mutable.ArrayBuffer.fill(tokens.size)(0.toDouble)
+    val bag = new Array[Double](tokens.size) //mutable.ArrayBuffer.fill(tokens.size)(0.toDouble)
```

Comment:
not sure what this comment is doing here. Most probably some leftover...

Reply:
Yup, seems like a leftover. Will remove